src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,21 @@
 Release History
 ===============
 
+2.34.1
+++++++
+
+**Compute**
```

Comment:
This should be App Service - are we putting this under compute now?

Reply:
Thank you for your reminder!
I also revised the title in release notes https://github.com/MicrosoftDocs/azure-docs-cli/pull/2998/files#r818235506